tests/cmake_test.cc

```diff
@@ -0,0 +1,18 @@
+#include <glib.h>
+#include "cmake.h"
+#include "project_build.h"
+#include <boost/filesystem.hpp>
+
+int main() {
+  CMake cmake(JUCI_TESTS_PATH);
+  
+  g_assert(cmake.project_path.filename()=="jucipp");
+  
```

Comment:
I don't want to force the users to name their source directory jucipp either. :P